cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/util/OAuthUtils.java

```diff
@@ -85,4 +85,18 @@ public static OAuthRegisteredService getRegisteredOAuthService(final ServicesMan
         }
         return null;
     }
+
+    /**
+     * Url encode a value.
+     *
+     * @param value the value to encode
+     * @return the encoded value
+     */
+    public static String urlEncode(final String value) {
```

Comment:
Even better. Let's collect them all in one spot. Makes it easier to troubleshoot issues later.